docs/source/gcp/examples/appengine-domain-mapping.rst

```diff
@@ -0,0 +1,24 @@
+App Engine - Audit domain mappings being added
+===============================================
+Custodian can audit new mappings being added. Note that the ``notify`` action requires a Pub/Sub topic to be configured.
```

Comment:
this would need to check the actual value of the domain to be useful imo, ie domain not in a white list.

Reply:
actually its a bit unclear to me if this mapping thing should be a resource, given the domain example below uses a different resource. reflecting every rest api resource into custodian isn't generally useful. i'm holding on a c7n_gcp release, because it feels like we've accumulated a bunch of resources that need to be reviewed on the basis.

Reply:
> ie domain not in a white list
This is actually checked in `gcp.app-engine-domain` that reflects only lower-level domains instead of the complete name. A possible way to check is using a regex which is not really convenient.
> is there a real world use case for this resource?
Considering possibility to extend the functionality of `gcp.app-engine-domain` and `gcp.app-engine-domain-mapping` with actions, there is: once found blacklisted domains in use, trigger deletion of corresponding domain mappings.

Reply:
domain mapping should be a filter on domain, that doesn't need to be addressed in this pr, but I'd like to avoid making new docs that need removal.